examples/python/aoa.py

```diff
@@ -0,0 +1,98 @@
+# Originally developed by JSBSim Team
+# Modified by Guilherme A. L. da Silva - aerothermalsolutions.co
+# Calculation required by aircraft icing enginering
+
```

Comment:
May be you should add to the comments that this script is designed to be executed from the folder which contains the jsbsim module or that the module should be installed via 'pip' or 'conda' ?